lib/babe/build.go

```diff
@@ -196,7 +196,7 @@ func (b *BlockBuilder) buildBlockExtrinsics(slot Slot, rt ExtrinsicHandler) []*t
 
 		err = determineErr(ret)
 		if err != nil {
-			logger.Warnf("failed to apply extrinsic %s: %s", extrinsic, err)
+			logger.Warnf("error when applying extrinsic %s: %s", extrinsic, err)
```

Comment:
why? It's the same thing as before

Reply:
I changed this because there is a case above that has the exact same log, but it continues rather than handle it. I changed the logs to differentiate between the two cases. But can leave if this is what we want.

Reply:
maybe

```suggestion
			logger.Warnf("determining apply extrinsic call error: %s", err)
```